cloudhub-2/modules/ROOT/pages/ch2-vpn.adoc

```diff
@@ -0,0 +1,143 @@
+
+Anypoint VPN supports site-to-site Internet Protocol security (IPsec) connections. A physical or software appliance, called a VPN endpoint, is the terminator on your side of the connection. The MuleSoft side of the connection is an implementation of a virtual private gateway (VGW). The MuleSoft VGW is associated with a single private space but can support up to 10 VPN connections.
+
+[NOTE]
+MuleSoft implementation capabilities may vary from other VGW offerings.
+
+== Anypoint VPN Features
+
+Each Anypoint VPN connection consists of two tunnels that enable you to connect to a single public IP address at a remote location. To configure connectivity to an additional public IP address at a remote location, you must create two VPN connections.
+
+A single VPN provides increased availability for your private space. Routine maintenance can briefly disable one of the two tunnels of your VPN connection. During this time, your VPN connection automatically fails over to the second tunnel so access is not interrupted. For this reason, both tunnels must be configured on your endpoint. Tunnel selection depends on your VPN endpoint capabilities and the routing type selection.
+
+The MuleSoft VGW implementation supports a maximum throughput of 1.25 Gbps. Multiple VPN connections to the same private space share the throughput capabilities of a single VGW. The VPN connection throughput depends on several factors, such as the capability of your VPN endpoint, the capacity of the connection, the average packet size, the protocol, and network latency between the gateways.
+
+=== IPsec Settings
+
+Anypoint VPN supports any combination of the following IPsec settings:
+
+* IKE version 1
+* IKE version 2 for route-based VPNs only
+* AES 128 or 256-bit encryption operating in the CBC and GCM
+* SHA or SHA-2 (256, 384, 512) hashing
+* Diffie-Hellman Phase 1 groups 2, 14-24
+* Diffie-Hellman Phase 2 groups 2, 5, 14-24
+* Timeouts for IKEv1
+** IKE (Phase 1) Lifetime: 28800 seconds (8 hours)
+** IPsec (Phase 2) Lifetime: 3600 seconds (1 hour)
+* Timeouts for IKEv2
+** IKE (Phase 1) Lifetime: 28000 seconds (7 hours and 50 minutes)
+** IPsec (Phase 2) Lifetime: 3000 seconds (50 minutes)
+
+[NOTE]
+Some of these IPsec settings are not supported in Government Cloud. For specific Government Cloud VPN settings, see xref:gov-cloud::gov-cloud-anypoint-vpn.adoc[Anypoint VPN Settings]. 
+
+=== Anypoint VPN Routing
+
+Anypoint VPN supports dynamic or static routing for VPN connections.
+
+* *Dynamic routing* Your device uses Border Gateway Protocol (BGP) to advertise routes to Anypoint VPN. Use BGP routing if your device supports this protocol.
+* *Static routing* - Requires you to specify the routes (subnets) in your network that are accessible through Anypoint VPN. +
+[NOTE]
+A maximum of 95 route table entries is permitted per private space, regardless of the number of VPN connections. Consolidate networks to the fewest number possible to avoid exceeding the limit.
```

Comment:
```suggestion
A maximum of 95 route table entries are permitted per private space, regardless of the number of VPN connections. Consolidate networks to the fewest number possible to avoid exceeding the limit.
```

I don't know which one is correct. I guess it could be singular if referring to the maximum, but plural if referring to the entries. I think "are" sounds better though.